app/assets/javascripts/admin/addon/components/admin-report.js

```diff
@@ -317,10 +321,10 @@ export default Component.extend({
       currentMode,
       options: this._buildOptions(currentMode, report),
     });
-  },
+  }
 
   _fetchReport() {
-    this._super(...arguments);
+    undefined;
```

Comment:
LOL

Reply:
Amazingly, this is 'technically correct'. If it had transformed it to `super._fetchReport()`, then it would raise an error like "super._fetchReport is not a function".

`this._super` usage when the method does exist on the superclass are transformed correctly.

So, I guess I'll just delete this line manually to clean up 🧹